ONSdigital/eq-author-app
# Specify the api proxy location once

### What is the context of this PR?
When running multiple versions of author in dev (dev env and a prototype) it requires running them on different ports. To specify the api proxy location current requires changing `author/package.json` and the start script. 

This change removes the need for the script.

### How to review 
1. Ensure `yarn start` works
1. Ensure you can run multiple author environments. This requires changing `author/package.json` and `api/docker-compose.yml` to specify the new ports.
